lib/active_merchant/delete_with_body.rb

```diff
@@ -0,0 +1,10 @@
+require 'net/http'
+require 'net/https'
+
+module ActiveMerchant
```

Comment:
Not gonna act like I'd build a library that uses class constants to define functionality, but at least it's convenient 👍